index.html

```diff
@@ -47,8 +47,8 @@ <h2> Haiku </h2>
 <footer> 
     
     <hr>
-        <blockquote>"The successeful Warrior is the average man with laser focus." <p> <i> &mdash; Bruce Lee </i> </p> </blockquote>
-        <p>&copy; Adam Griffith</p>
+        <blockquote>"The successeful Warrior is the average man with laser focus." <br><i> &mdash; Bruce Lee </i></blockquote>
```

Comment:
`<br>` is probably more appropriate here than a whole new paragraph for an author attribution (but no marks lost for this)